codespell_lib/data/dictionary.txt

```diff
@@ -18372,10 +18373,15 @@ obstacal->obstacle
 obstancles->obstacles
 obstruced->obstructed
 obsure->obscure
+obtaien->obtain
```

Comment:
Grep.app doesn't find any hits for this, I think because what it's presumably based on `obtaiend` is a result of swapping two of the end characters, whereas this is about inserting another character and the e is no-where near the i or n.

Reply:
There are a few available via e.g. google though.

Typo in a code comment:
https://stackoverflow.com/questions/35522401/updating-jagged-array-without-losing-the-value-in-previous-rows-in-c-sharp

Some sort of academical stuff:
https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM1538107
https://www.researchgate.net/publication/257058426_Prevencion_del_cancer_de_cervix_Comparacion_de_la_calidad_de_la_muestra_segun_utilicemos_bastoncillo_o_cepillo_cervical
https://www.icao.int/SAM/Documents/2015-SAMAIM7/SAMAIM7_WP07_SistAutomat.pdf

Not sure if these hits justify the addition of this typos.

Reply:
> Typo in a code comment:
> https://stackoverflow.com/questions/35522401/updating-jagged-array-without-losing-the-value-in-previous-rows-in-c-sharp

Grammar wise, this one looks like it should probably be obtained.

> Some sort of academical stuff:
> https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM1538107

Agreed
> https://www.researchgate.net/publication/257058426_Prevencion_del_cancer_de_cervix_Comparacion_de_la_calidad_de_la_muestra_segun_utilicemos_bastoncillo_o_cepillo_cervical

Looks like it should perhaps be obtained again
> https://www.icao.int/SAM/Documents/2015-SAMAIM7/SAMAIM7_WP07_SistAutomat.pdf

Agreed

Okay, personally I'm sold, on the proviso you add obtained too as a second correction.

Reply:
Thanks for your second review / opinion about that. I have added `obtained` as a second correction in b0392bc9713a89c56f0815ecad5dc1aa67db2a1a

Reply:
I'd have been happy for obtain to be the primary correction, but it's probably about equal.